org.adempiere.base/src/org/compiere/process/SvrProcess.java

```diff
@@ -788,6 +790,8 @@ private void autoFillParameters(){
                     	toField.set(this, parameter.getParameter_ToAsInt());
                 } else if (type.equals(String.class)) {
                     field.set(this, (String) parameter.getParameter());
+                    if(notField != null)
+                    	notField.set(this, (boolean) parameter.isNotClause());

```

Comment:
it seems you are not passing the parameter value to the not field. can you provide an example of the annotation for this ?

Reply:
We can use annotations like https://wiki.idempiere.org/en/NF9_Annotated_Process_Parameters
An example:

@Parameter
   private String p_GL_Category_ID;
@Parameter
  private Boolean p_GL_Category_NOT;